src/checks/optional_product_selection.ts

```diff
@@ -5,13 +5,14 @@ import { it, page } from "../lib/helpers";
 // otherwise it expects the initial root password dialog
 export function optionalProductSelection(productName: string) {
   it("should optionally display the product selection dialog", async function () {
-    // Either the root password setting is displayed or there is
-    // the product selection page.
+    // Either the overview is displayed or there is the product selection page.
     const productSelectionDisplayed: boolean = await Promise.any([
-      page.waitForSelector("input#rootPassword").then((s) => {
+      // left navigation item present
+      page.waitForSelector("a[href='#/overview']").then((s) => {
```

Comment:
NP: I know almost nothing about this tests, but I wonder if it is not possible / easier / more reliable to look for "Overview" heading or "Select a product" text. Although "Select a product" is certainly something not too stable since it should be "Choose 1 of 7 available products" or similar.

Just a question